src/current/cockroachcloud/managed-backups.md

```diff
@@ -0,0 +1,382 @@
+{{site.data.alerts.end}}
+
+## Managed backup settings
+
+You can modify the settings of managed backups for [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters).
+
+[{{ site.data.products.basic }} clusters](#basic-clusters) have a default configuration that you cannot modify.
+
+### Basic clusters
+
+Cockroach Labs will take a managed backup every 24 hours. By default, managed backups will be retained for 30 days in {{ site.data.products.basic }} clusters.
+
+Once a cluster is deleted, Cockroach Labs retains the backup for 30 days. If an organization is deleted, you will lose access to all of the managed backups that Cockroach Labs has taken of the cluster after 30 days.
+
+For more details on restoring a managed backup, refer to the [Cloud Console](#restore-a-basic-cluster) section. To restore a backup from a deleted cluster, you must contact the [Cockroach Labs Support team](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/support-resources).
+
+### Standard and Advanced clusters
+
+In {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters you can configure the [frequency](#frequency) and [retention](#retention) of managed backups.
+
+For instructions on how to view and configure managed backup settings, use:
+
+- The [Cloud Console](#cloud-console) for {{ site.data.products.standard }} or {{ site.data.products.advanced }}.
+- The [Cloud API]({% link cockroachcloud/cloud-api.md %}) for [{{ site.data.products.standard }}](#cloud-api-for-standard-clusters) or [{{ site.data.products.advanced }}](#cloud-api-for-advanced-clusters).
+
+#### Frequency
+
+You can configure how frequently Cockroach Labs takes backups, which will determine the cluster's [RPO]({% link {{site.current_cloud_version}}/disaster-recovery-overview.md %}#resilience-strategy).
+
+You can set backup frequency to one of the following options:
+
+{% include cockroachcloud/backups/frequency-settings.md %}
+
+#### Retention
+
+You can set your retention duration **once**. After you have adjusted the retention, the duration will only apply to new backups. The available retention options are:
+
+{% include cockroachcloud/backups/retention-settings.md %}
+
+{% include cockroachcloud/backups/retention-deleted-cluster.md %}
+
+## Upgrades and downgrades
+
+If you have upgraded from a {{ site.data.products.basic }} cluster to a {{ site.data.products.standard }} cluster, the existing backup schedules will still apply, but you can then configure the frequency and retention of future managed backups in the {{ site.data.products.standard }} cluster.
+
+If you have downgraded from a {{ site.data.products.standard }} cluster to a {{ site.data.products.basic }} cluster, existing managed backups will be retained for the configured retention duration. The default managed backups in {{ site.data.products.basic }} clusters will be taken every 24 hours and have a 30-day retention.
+
```

Comment:
We have this issue where the retention period really only applies to the last full backup, because once the full backup expires we don't have a way to restore the incrementals. So if a retention period is set to 2d, it's really more like 1d because we take a full back every day. But we're also not exposing the full backup concept to users. I wonder if maybe we should expose the concept that "cockroach will take the correct combination of full/incremental backups on your behalf" otherwise it'll be hard to explain this.

Reply:
Just to provide some more info on this, if a full backup is taken once every `f`, and a retention period is set to `r`, then an incremental is useable anywhere between `r` and `r-f`.

For example, to alleviate this, we have decided to take full backups every day. So that means with a 30 day retention, the incrementals could be useable for anywhere between 30 days and 29 days. With a 7 day retention, the incrementals could be useable for anywhere between 7 days and 6 days.

If fulls were taken every 5 minutes, then with a 7 day retention, incrementals would be useable between 7 days and 6 days, 23 hours, and 55 minutes.

We don't want to be so specific in our documentation, but hopefully this guides the wording.

Reply:
yeah but fulls are not taken every 5minutes, I feel like we may need to be more explicit about that? what do you think?

Reply:
Oh what I mean about "not so specific" is we probably don't need to go into the math so much?

Given current limitations with full/incremental recurrence ratio, I think the highest full backup frequency we can do is 4 hours, and the lowest we are doing is capped at 1 day. So that means for some retention `r`, the incrementals could be useable for at least `r - 4 hours` or `r - 1 day`. But this is highly dependent on the full backup frequency, which customers cannot set separately.

Additionally, these numbers could be changing depending on how we set our full backup frequency (as of right now, the actual highest full backup frequency is once every hour with a 5 minute RPO). So I don't want the documentation to be too specific.

Reply:
So, are we taking full backups every 24 hours (whatever the frequency is). Or, might it be more often than that?

I'm trying to figure out how customers can identify the usable backups, and how I can explain that. If the full backup frequency is changeable, that's tough to reason about (particularly in Standard where the backup type is not labeled).

Reply:
I've added the two paragraphs for this that I sent to you both on Slack. Feel free to suggest edits to this still

Reply:
This looks good to me